decentraland/unity-renderer
# feat: animation on spawn and wearable change

## What does this PR change?

Feature requirement #2734 

Adds an animation for changing wearables in the AvatarEditorHud. There are different animations for upper, lower, shoes, and accessories wearables. 

Also adds a spawn animation that runs after the avatar hologram animation is complete.

## How to test the changes?

1. Open the backpack.
2. Change upper, lower, shows, and accessories wearables. Check that animation is run in each case.

**(STEPS DEPRECATED, not necessary to test. This will be done on other RFC)**
3. Save changes. See that another animation is run on reload in world.
4. Connect with a peer. Make the peer do some wearables changes. Check that the spawn animation is run on reload of the avatar peer.

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
